judy/src/App.js

```diff
@@ -0,0 +1,90 @@
+import logo from './logo.svg';
+import './App.css';
+import brioche from './pics/brioche.JPG';
+import canoeing from './pics/canoeing.jpeg';
+import cinbns from './pics/cin buns.jpeg';
+import hiking from './pics/hiking.jpeg';
+import painting from './pics/painting.jpeg';
+import strawbcake from './pics/strawb cake.jpeg';
+import walk from './pics/walk.jpeg';
+
+
+
+function App() {
+  const onChangeFunction = (event) => {
+    console.log(event.target.value);
+  }
+  
+  const buttonFunctionGit = () => {
+    window.open("https://github.com/judy-guo")
+    console.log("the 'explore my github' button was clicked");
+  }
+  const buttonFunctionVSCO = () => {
+    window.open("https://vsco.co/juds-foods/gallery")
+    console.log("the 'explore my vsco' button was clicked");
+  }
+  const activityList = ["hiking", "canoeing", "procrasti-baking"];
+ 
+  const activityMap = activityList.map((activity) => {
+    return(
+      <p>I enjoy {activity}.</p>
+    )
+  });
+ 
+ 
+  return (
+    <div>
+    <div className="App">
+      <header className= "App-contact">
```

Comment:
theres nothing technically wrong with using all of these `<header>` tags, but generally they are used for a singular header on the page. In your case you're looking more for an `<h1>` tag. But there was no way you could have known this.